incident.go

```diff
@@ -176,15 +176,21 @@ func (c *Client) ListIncidentNotes(id string) ([]IncidentNote, error) {
 
 // IncidentAlert is a alert for the specified incident.
 type IncidentAlert struct {
-	ID        string    `json:"id,omitempty"`
-	Summary	  string    `json:"summary,omitempty"`
-	CreatedAt string    `json:"created_at,omitempty"`
-	AlertKey  string    `json:"alert_key,omitempty"`
+	APIObject
+	CreatedAt   string       `json:"created_at,omitempty"`
+	Status      string       `json:"status,omitempty"`
+	AlertKey    string       `json:"alert_key,omitempty"`
+	Service     APIObject    `json:"service,omitempty"`
+	Body        interface{}  `json:"body,omitempty"`
```

Comment:
I don't know how to deal with that not working field.

It has various of data even a key that are not represented on the official API documentation like `cef_details`.

I would like to store it as a "Various mixed array", but I don't know how to do to make json unmarshal happy.

Tried `interface{}` and `[]interface{}` without success:

```
json: cannot unmarshal number into Go value of type []pagerduty.IncidentAlert
```

Any help would be appreciated.